app/models/katello/erratum.rb

```diff
@@ -46,10 +46,8 @@ def self.repository_association_class
     end
 
     def self.applicable_to_hosts(hosts)
-      self.joins(:content_facet_errata).joins(:content_facets).
-         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).
-         select("DISTINCT ON (#{self.table_name}.updated, #{self.table_name}.id) #{self.table_name}.*").
-         order("#{self.table_name}.updated desc, #{self.table_name}.id")
+      self.joins(:content_facets).
+        where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).uniq
```

Comment:
Rather than testing for slowness which seems a bit arbitrary, it might be better to test the performance and ensure that performance remains the same as the Katello project moves forward. Minitest has benchmarking tools for asserting the continued performance of chunks of code (``assert_performance_constant``, ``assert_performance_linear``, ``assert_performance_logarithmic``, ``assert_performance_exponential``, and ``assert_performance_power``). I'm not quite sure how we could apply that to our questionable DB queries, but there are tools that we could start (as @beav so wonderfully put it) noodle with.